package.json

```diff
@@ -1,38 +1,38 @@
 {
-  "name": "bssr-nodes-bot",
-  "version": "1.0.0",
-  "description": "Main bot for BSSR Nodes.",
-  "main": "index.js",
-  "scripts": {
-    "start": "npm install && node index.js"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/bssr-nodes/bssr-nodes-bot.git"
-  },
-  "author": "DanBot Hosting LTD",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/bssr-nodes/bssr-nodes-bot/issues"
-  },
-  "homepage": "https://github.com/bssr-nodes/bssr-nodes-bot#readme",
-  "dependencies": {
-    "@discordjs/rest": "^2.3.0",
-    "ansi-to-html": "latest",
-    "axios": "^1.4.0",
-    "better-sqlite3": "^8.5.0",
-    "chalk": "^3.0.0",
-    "discord-api-types": "^0.37.92",
-    "discord.js": "^14.16.2",
-    "humanize-duration": "^3.28.0",
-    "moment": "^2.29.4",
-    "ms": "^2.1.3",
-    "nodemailer": "^6.9.3",
-    "os-utils": "0.0.14",
-    "path": "^0.12.7",
-    "ping-tcp-js": "^1.5.0",
-    "quick.db": "^7.1.3",
-    "strip-ansi": "latest",
-    "validator": "^13.9.0"
-  }
-}
\ No newline at end of file
+	"name": "bssr-nodes-bot",
+	"version": "1.0.0",
+	"description": "Main bot for BSSR Nodes.",
+	"main": "index.js",
+	"scripts": {
+		"start": "npm i && node ."
```

Comment:
We don't use this to start the bot and that so not really needed

Reply:
It was already there, so I didn't really bother to remove it.

Reply:
Probably from dbh bot, never noticed it, never used it

Reply:
Alright.